sale_order_lot_selection/wizards/stock_lot_add_to_sale_order.py

```
@@ -0,0 +1,161 @@
+        "res.partner",
+        string="Customer",
+        store=True,
+        compute="_compute_partner_id",
+    )
+
+    @api.depends("sale_order_id")
+    def _compute_partner_id(self):
+        for record in self:
+            record.partner_id = record.sale_order_id.partner_id.id
+
+    def open_wizard(self, lot_ids):
```

Ruff (0.8.2)

Missing return type annotation for public function `open_wizard`

(ANN201)